src/woo_publications/templates/admin/document_change_list.html

```diff
@@ -0,0 +1,32 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_urls static admin_list django_admin_index %}
+
+{% block extrastyle %}{{ block.super }}
+<link rel="stylesheet" href="{% static "admin/css/admin-index.css" %}">{% endblock %}
+
+{% block nav-global %}{% include "django_admin_index/includes/app_list.html" %}{% endblock nav-global %}
+
+{% if not is_popup %}
+{% block breadcrumbs %}
+{% admin_index_settings as admin_index_settings %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+
+{# NOTE: Remove app groups if needed #}
+{% if not admin_index_settings.HIDE_APP_INDEX_PAGES %}
+&rsaquo;
+<a href="{% url 'admin:app_list' app_label=cl.opts.app_label %}">{{ cl.opts.app_config.verbose_name }}</a>
+{% endif %}
+{% block breadcrumbs_pre_changelist %}{% endblock %}
+&rsaquo; {{ cl.opts.verbose_name_plural|capfirst }}
+</div>
+{% endblock %}
+{% endif %}
+
+{% block object-tools-items %}
+    <li>
+        <a href="{% url 'admin:publications_publication_changelist' %}">{% trans 'publicaties' %}</a>
+    </li>
```

Comment:
this is not what was intended - instead in the publications admin there should be a link pointing to the document change list which filters on that particular publication, so you can quickly get a list of related documents for a given publication. so the relation is the other way around